src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGalleryListView.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+
+import {
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  DataListAction,
+  Flex,
+  FlexItem,
+  Label,
+  LabelGroup,
+} from '@patternfly/react-core';
+import OutlinedQuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+import { useTranslation } from '@rhoas/app-services-ui-components';
+
+export type ConnectorTypesListItemProps = {
+  id: string;
+  labels: string[];
+  title: string;
+  verison: string;
+};
+
+export const ConnectorTypeListItem: React.FunctionComponent<ConnectorTypesListItemProps> =
+  ({ id, labels = [], title, verison }) => {
+    const [selectedDataListItemId, setSelectedDataListItemId] =
+      React.useState('');
+
+    const onSelectDataListItem = (id) => {
+      setSelectedDataListItemId(id);
+    };
+
+    const handleInputChange = (
+      id: string,
+      _event: React.FormEvent<HTMLInputElement>
+    ) => {
+      setSelectedDataListItemId(id);
+    };
+
+    const { t } = useTranslation();
+
+    return (
+      <DataList
```

Comment:
How does this look without this component wrapping the `DataListItem`?

Reply:
It looks the same without the wrapping.

Reply:
Cool!  One thing that could be interesting to try is to change [this line](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGalleryViewport.tsx#L164) and pass `DataList` here, but I guess if these components render okay without it, then :shrug: